src/utils/InstabugConstants.ts

```diff
@@ -11,6 +11,7 @@ const InstabugConstants = {
     'IBG-RN: Expected key and value passed to setUserAttribute to be of type string',
   REMOVE_USER_ATTRIBUTES_ERROR_TYPE_MESSAGE:
     'IBG-RN: Expected key and value passed to removeUserAttribute to be of type string',
+  DEFAULT_METRO_SERVER_URL: '8081',
```

Comment:
I believe this
```suggestion
  DEFAULT_METRO_PORT: '8081',
``` 
would make more sense here what do you think ?